src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * deprecated using `int` or `float` as data for the `NumberType` when the `input` option is set to `string`
+ * Render accept attribute for `FileType` if mime types is used in assert validator
```

Comment:
The type guesser guesses the HTML `accept` attribute when a mime type is configured in the `File` or `Image` constraint.